bundle/next/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -275,13 +269,17 @@ spec:
       identityProviderRealm: 'myrealm'
       identityProviderClientId: 'myClient'
       ```
-    #### Self-signed Certificates and TLS Secrets
-
-    Eclipse Che uses self-signed certificates by default, as TLS mode is on by default.
 
-    To provide the name of the tls secret that will be used for Eclipse Che and workspaces ingresses:
+    #### Self-signed Certificates and TLS Secrets
+    Eclipse Che uses auto-generated self-signed certificates by default and TLS mode is on.
+    If it is needed to use your own certificates, create a new secret:
+    ```
+    kubectl create secret tls my-tls-secret --key <PATH_TO_PRIVATE_KEY_CERTIFICATE> --cert <PATH_PUBLIC_KEY_CERTIFICATE> -n my-eclipse-che
```

Comment:
It still doesn't show what structure the secret should have, what the relations between the fields are.
I still think that this is matter of docs, but not this sort description.
Since we have no such docs, I'd suggest to remove this two kubectl commands at all (but ideally to create the docs and put the link here).

Reply:
agree